Streetcode/Streetcode.WebApi/Extensions/SeedingLocalExtension.cs

```diff
@@ -35,8 +35,8 @@ public static async Task SeedDataAsync(this WebApplication app)
                 string blobPath = app.Configuration.GetValue<string>("Blob:BlobStorePath");
                 var repo = new RepositoryWrapper(dbContext);
                 var blobService = new BlobService(blobOptions, repo);
-                string initialDataImagePath = "../Streetcode.DAL/InitialData/images.json";
-                string initialDataAudioPath = "../Streetcode.DAL/InitialData/audios.json";
+                string initialDataImagePath = "../../../TestData/InitialData/images.json";
+                string initialDataAudioPath = "../../../TestData/InitialData/audios.json";
```

Comment:
Maybe create some sort of variable for path? Just a suggestion, maybe it is better this way